### ISSUE TITLE
If only the onset dataset is trained, how accurate will it be

### ISSUE THREAD

**Issue by water202232** [jongwook/onsets-and-frames#33]
I used TensorFlow Keras to implement onset_stack neural network, and make the corresponding training data. According to the understanding of this model, the onset_stack is just the starting point of piano sound training. But I don't know why. The accuracy of my training is extremely low. After several hours of training, the accuracy rate is only about 0.05. Please ask, can't we only train the onset dataset? If only the onset dataset is trained, what is the accuracy will be?
